src/governance-vote.h

```diff
@@ -197,6 +207,8 @@ class CGovernanceVote
 };
 
 
+bool operator==(const CGovernanceVote& vote1, const CGovernanceVote& vote2);
```

Comment:
Also, it's common (for current code) to include implementation for operators in header file. Probably a good idea to follow, but I have no strong opinion on that one.